aca_lts_eval.py

```diff
@@ -176,10 +177,45 @@ def select_ri_guide_stars(ra, dec, cone_stars):
     return G_RI_CAT_CACHE[id_key]
 
 
-def t_lose_star(mag, ref_mag=10.3, ref_tccd=-11.5, scale_4c=None):
+def guide_count(mags, tccd=-10.2):
+    """
+    Given mags from guide stars and a temperature, calculate a guide star
+    count using signal-to-noise scaled mag thresholds.
+    """
+    thresh1 = snr_mag_for_tccd(tccd, ref_mag=10.0)
```

Comment:
These need to include a comment like
```
# The reference mags and corresponding fractional star count are defined
# for a reference temperature of -10.2 degC.  This was discussed and agreed
# to by SSAWG on DATE.
thresh0 = snr_mag_for_t_ccd(t_ccd, ref_mag=6.0, ref_t_ccd=-10.2)
...
```

Reply:
And actually now that I think about it, the standard ref temperature is whatever the planning limit was when we agreed on this metric at SSAWG (unless we specifically picked a different temperature, but I can't find any relevant notes).  I believe this was -10.9 since the guide stars fractional counts notebook I did was presented March 21 this year and the limit increase was after that.

Reply:
I thought we presented/used 10.3 @ -11.5 at the state of the aca.  We probably agreed on the metrics at -10.9C.  And then SSAWG didn't put a lien on updating the metrics to move to a planning limit of -10.2C, which could be considered approval.

Do you want to just pick one (it sounds like -10.9 was the way you were going)?  Or do you want me to run the eval tool both or all three ways and look at a few observations in the grey area?

Reply:
Regarding "# The reference mags and corresponding fractional star count are defined
# for a reference temperature of -10.2 degC.  This was discussed and agreed
# to by SSAWG on DATE."
I can't find any reference of when this was agreed to, but I'll assume that we agreed on these limits with the presentation of your fractional count notebook showing count "impact".